src/pybamm/solvers/base_solver.py

```diff
@@ -140,8 +140,10 @@ def set_up(self, model, inputs=None, t_eval=None, ics_only=False):
             model.calculate_sensitivities = []
 
         # see if we need to form the explicit sensitivity equations
+        # Skipping JaxSolver doesn't currently support sensitivities
         calculate_sensitivities_explicit = (
-            model.calculate_sensitivities and not isinstance(self, pybamm.IDAKLUSolver)
+            model.calculate_sensitivities
+            and not isinstance(self, (pybamm.IDAKLUSolver, pybamm.JaxSolver))
```

Comment:
We've started using properties to encode solver features (e..g see `supports_parallel_solve`). Can you make this a property of the solver so you can just say ` model.calculate_sensitivities and not self.requires_explicit_sensitivities()`

Reply:
Sure, I'll take a look. I was wonder if we should be raising an error here instead. As the`calculate_sensitivities` argument is essentially ignored by the pure Jax solvers. Do you have an opinion either way?